Dockerfile.ubi8

```
@@ -0,0 +1,49 @@
+FROM golang:1.18.1 AS build
+WORKDIR /go/src/github.com/aquasecurity/kube-bench/
+COPY makefile makefile
+COPY go.mod go.sum ./
+COPY main.go .
+COPY check/ check/
+COPY cmd/ cmd/
+COPY internal/ internal/
+ARG KUBEBENCH_VERSION
+RUN make build && cp kube-bench /go/bin/kube-bench
+
+
+# ubi8-minimal base image for build with ubi standards
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5 as run
+
+RUN microdnf install yum \
```

`findutils` is needed for having `find` commands and`openssl` is needed for having `openssl` commands

```suggestion
RUN microdnf install yum findutils openssl \
```

---

added findutils and openssl